src/osa/scripts/sequencer.py

```diff
@@ -205,6 +206,36 @@ def update_sequence_status(seq_list):
             )
             seq.muonstatus = int(Decimal(get_status_for_sequence(seq, "MUON") * 100) / seq.subruns)
             seq.dl2status = int(Decimal(get_status_for_sequence(seq, "DL2") * 100))
+            seq.catbstatus = check_catB_status(seq)
+
+
+def check_catB_status(seq):
+    catbstatus = 'None'
+    if seq.type == "DATA":
+        directory = options.directory
+        # Buscar archivos "closed"
+        closed_files = list(directory.glob(f"catB*{seq.run}*.closed"))
+        if any(f.exists() for f in closed_files):
+            catbstatus = 'CLOSED'
+        else:
+            log_files = list(options.log_directory.glob(f"catB_calibration_{seq.run}_*.err"))
+
+            if log_files and any(f.exists() for f in log_files):
+                status_set = False
+                for f in log_files:
+                    if f.exists():
+                        with f.open("r") as fh:
+                            for line in fh:
+                                if "CRITICAL" in line:
+                                    seq.catbstatus = 'FAILED'
+                                    status_set = True
+
+
+                # Si ningún log tenía CRITICAL/ERROR pero existen archivos
+                if not status_set and any(f.exists() for f in log_files):
+                    catbstatus = 'RUNNING'
```

Comment:
The same here, if you get the SLURM job id, then using get_sacct_output(run_sacct(job_id=job_id)) you can get its status.